src/Server.h

```diff
@@ -144,6 +152,9 @@ class cServer
 	from the settings. */
 	bool ShouldAllowMultiWorldTabCompletion(void) const { return m_ShouldAllowMultiWorldTabCompletion; }
 
+	/** Get the Forge mods registered for a given protocol. */
+	AStringMap & GetRegisteredForgeMods(const UInt32 a_Protocol);
```

Comment:
If I read it right, RegisterForgeMod and UnregisterForgeMod can directly access m_ForgeModsByVersion, so they can modify that directly and GetRegisteredForgeMods can still return a const, preventing other callers from modifying it.